### DISCUSSION TITLE
ERROR  Your application tried to access @nuxt/kit

### DISCUSSION THREAD

**Discussion by nyxb** [nuxt/nuxt#17868] [Questions] [answered]
With last stable yarn now its version 3.2.0 not all depencies get install it gets following error 

 ERROR  Your application tried to access @nuxt/kit, but it isn't declared in your dependencies; this makes the require call ambiguous and unsound.

**Comment by danielroe** [comment, marked as answer]
Ensure you're using the node-modules linker, which works fine. For PNP support follow https://github.com/nuxt/framework/issues/581.

**Reply by nyxb** [reply]
> Ensure you're using the node-modules linker, which works fine. For PNP support follow https://github.com/nuxt/framework/issues/581.

you are my man, thank you for all the quick replies and the great support. excellent ⭐⭐⭐
